workshops/weather_app/README.md

```diff
@@ -0,0 +1,416 @@
+
+Lastly, we will add `<div>` with class as `main-weather` to display the weather details.
+
+We will be using `<p>` for every detail we display. Hence we declare the weather details with the following ids
+
+```html
+<p id="date">Date </p>
+<p id="city">City </p>
+<p class="temp" id="temp">Temp </p>
+<p id="min-max">Min and Max Temp </p>
+<p id="weather-type">Sunny </p>
+```
+
+Our `HTML` code is done and now we can move to the `CSS` part. At the end, the `HTML` code will look like this:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.0/moment.min.js"></script>
+	<title>Weather</title>	
+</head>
+
+<body>
+	<div class="container">
+ 		<div class="content">
+    		
+    		<h1>WEATHER APP</h1>
+        	<input id="input" class="input" placeholder="Enter the City Name">
+    	</div>
+
+    <div class="main-weather">
+    	    <p id="date">Date </p>
+            <p id="city">City </p>
+            <p class="temp" id="temp">Temp </p>
+            <p id="min-max">Min and Max Temp </p>
+            <p id="weather-type">Sunny </p>
+    </div>
+</div>
+
+<script src="script.js"></script>
+
+</body>
+</html>
+```
+
+## 4. Improving the design of skeleton using CSS
+
+We will customize the body by giving it background colour, display property, font-size and line-height.
+
+We will set display value as `flex` because Flexbox makes it easier to design a flexible responsive layout without using float or positioning.
+
+Till now the code would look something like this:
+
+```css
+body {
+  background-color: #dfe7ee;
+  line-height: 1.5;
+  font-size: 125%;
+  display: flex;
+}
+```
+
+Now we will start making the Card.
+
+```css
+.container {
+  background-color: #fff;
+  padding: 0 4.5em 7em; 
+  margin: 100px 500px 400px auto; 
+  -webkit-filter: drop-shadow(0 1em 1em rgba(0, 0, 0, 0.1));
+          filter: drop-shadow(0 1em 1em rgba(0, 0, 0, 0.1));
+}
+```
+
+`WebKit-filter` with `drop-shadow` is a very cool property which is used for giving a highlighted shadow behind the card.
+
+We will add some properties to our heading by adding the following code:
+
+```css
+h1 {
+  border-bottom: 4px solid deepskyblue;
+  padding-bottom: 0.25em;
+  margin-bottom: 1em;
+  text-align: center;
+  font-family: Raleway;
+}
+```
+ 
+Till this point, we have designed almost all the elements and are only left with the input box and the details which we wish to display.
+ 
+For the input box we add the following code:
+ 
+ ```css
+ .input {
+    border: none;
+    outline: none;
+    font-size: 1.4rem;
+    text-align: center;
+    font-weight: bold;
+}
+```
+And for the weather details, we add the following code:
+
+```css
+.main-weather {
+    display: none;
+    line-height: 2.2rem;
+    height: 30vh;
+    text-align: center;
+    color:  #23313E;
+    font-weight: bold;
+}
+
+.temp { 
+    margin: 25px;
+    font-size:40pt;
+}
+

```

Comment:
don't just have me copying code I don't know

Reply:
`margin, font-size` are some basic property so I don't feel that it should be explained!